bin/cli.js

```diff
@@ -267,6 +267,47 @@ function invoke(env) {
         .catch(exit);
     });
 
+  commander
+    .command('list')
+    .alias('status')
+    .description('        List all migrations files with status.')
+    .action(() => {
+      pending = initKnex(env, commander.opts())
+        .migrate.list()
+        .then(([completed, newMigrations]) => {
```

Comment:
Would be great if you could extract whole logic for this operation into a separate file and only call it from cli.js